src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -568,7 +571,8 @@ private String getStudentHomePageViewLink(String googleId) {
     }
 
     public String getLogInfoForTableRowAsHtml() {
-        return "<tr" + (isFirstRow ? " id=\"first-row\"" : "") + "> <td class=\"" + getTableCellColorCode(timeTaken) + "\" style=\"vertical-align: middle;\">"
+        return "<tr" + (isFirstRow ? " id=\"first-row\"" : "") + ">"
+               + "<td class=\"" + getTableCellColorCode(timeTaken) + "\" style=\"vertical-align: middle;\">"
```

Comment:
Is the missing space before `<td class>` intentional?

I realized there are spaces between the closing tags at lines 590 and 591 too